kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/README.md

```diff
@@ -384,3 +384,40 @@ Sometimes in production, but most likely in testing environments, you might want
 $ kubectl create -f delete-domain-job.yaml
 
 ```
+## Troubleshooting
+
+1. Message: "status on iteration 20 of 20
+pod domain1-create-weblogic-sample-domain-job-4qwt2 status is Pending
+The create domain job is not showing status completed after waiting 300 seconds."  
+* The most likely cause is related to the value of persistentVolumeClaimName, defined in domain-home-on-pv/create-domain-inputs.yaml.  
+To find out if this is the problem, do the following:
+    * execute `kubectl get all --all-namespaces` to find the name of the create weblogic sample domain job.
+    * execute  `kubectl describe pod <name-of-create-weblogic-sample-domain-job>` to see if there is an Event that has text similar to `persistentvolumeclaim "domain1-weblogic-sample-pvc" not found`
+    * find the name of the pvc that was created by executing [create-pv-pvc.sh](../../create-weblogic-domain-pv-pvc/README.md), using `kubectl describe pvc`. It is likely to be `weblogic-sample-pvc`
+
+Change the value of persistentVolumeClaimName to match the name created when you executed [create-pv-pvc.sh](../../create-weblogic-domain-pv-pvc/README.md). Rerun the `create-domain.sh` script with the same arguments as you did before.
```

Comment:
Probably should recommend deleting the failed domain's resources before rerrunning, and include a reference to the delete-domain README...

Reply:
actually, the extra steps are not necessary in this particular situation.  Change and rerun is all I needed to do when I tried this the first time, and was all I needed to do when I retried the test, just now.